service/src/main/kotlin/io/provenance/explorer/domain/entities/Transactions.kt

```diff
@@ -589,3 +595,76 @@ class TxEventAttrRecord(id: EntityID<Int>) : IntEntity(id) {
     var attrKey by TxEventAttrTable.attrKey
     var attrValue by TxEventAttrTable.attrValue
 }
+
+object TxSingleMessageCacheTable : IntIdTable(name = "tx_single_message_cache") {
+    val txTimestamp = datetime("tx_timestamp")
```

Comment:
Should add `processed` here as a column.

Reply:
Changed to add new column.